docs/integrations/sources/chargebee.md

```diff
@@ -99,7 +99,8 @@ The Chargebee connector should not run into [Chargebee API](https://apidocs.char
 
 | Version | Date       | Pull Request                                             | Subject                                                                                             |
 | :------ | :--------- | :------------------------------------------------------- | :-------------------------------------------------------------------------------------------------- |
-| 0.4.1   | 2024-03-03 | [35509](https://github.com/airbytehq/airbyte/pull/35509) | Updates CDK version to latest (0.67.1), updates `site_migration_detail` record filtering                                                   |
+| 0.4.2   | 2024-03-14 | [36037](https://github.com/airbytehq/airbyte/pull/36037) | Adds fields to stream schemas                                                                       |
```

Comment:
nit: Extended the schemas with missing fields for: x, y, z streams

Reply:
Added